packages/flutter/lib/semantics.dart

```diff
@@ -0,0 +1,25 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// The Flutter rendering tree.
```

Comment:
This doc comment probably needs to be updated to talk about semantics instead of rendering?

Reply:
Good catch, I wrote a new comment, see if it makes sense